vignettes/model_definition.Rmd

```diff
@@ -18,22 +18,9 @@ vignette: >
-
-The method requires at least one observation, at delay $d=0$ for the most recent reference date, located at the bottom of the reporting triangle in Fig 1 above. The method assumes that the values at each delay $d$ column will be the same proportion of the values previously reported. To fill in the missing values for each column $d$, we sum over the rectangle of completely observed reference dates for all $d-1$ columns (block top left) and sum over the column of completely observed reference dates for all of the entries in column $d$ (block left). The ratio of these two sums is assumed to be the same in the missing entries in column $d$, so we use the entries observed up to $d-1$ for each incomplete reference date (block bottom left), and scale by this ratio to get the missing entries in column $d$. This process is repeated for each delay from $d$ to the maximum delay $D$. At each iteration an additional reference date entry is computed as the delay $d$ increases.
-
-# Mathematical model
-
-The following describes the estimate of the delay distribution, the generation of the point nowcast, and the estimate of the observation error, for a partially observed or complete reporting triangle.
-The method itself is time-unit agnostic, which means it can be used to generate nowcasts for any temporal granularity in the reporting and reference of data.
-In the case where the temporal granularity of the reporting differs from the referencing, e.g.
-weekly releases of daily data, pre-processing will need to be done so that the units of the delays $d$ and the units of the reference time $t$ are the same (e.g. by filling in 0s for all the days of the week that no data is reported).
-This method is based on the method described by [Wolffram et al. 2023](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011394).
+The `baselinenowcast` model, initially developed as a reference for the COVID-19 hospital admissions nowcasting challenge in Germany (2021-2022, [Wolffram et al](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011394), utilises reporting triangles of preliminary case counts and their delays. It applies a multiplicative approach, using empirically observed historical delay distributions to estimate yet-to-be-observed cases.
+Users can specify whether delay distributions are estimated from the current reporting triangle, fully observed past data, or data from different strata. The model produces point nowcasts by "filling in" the reporting triangle.
```

Comment:
I think we want to decide if a point nowcast = completed reporting square or if it is the sum of point estimates at each element in completed reporting square